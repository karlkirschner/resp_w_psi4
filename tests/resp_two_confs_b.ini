[molecules]
input_files: ethanol_1.xyz, ethanol_2.xyz

[vdw.surface.options]
vdw_scale_factors : 1.4, 1.6, 1.8, 2.0
vdw_point_density : 1.0
esp               : ethanol_1_esp.dat, ethanol_2_esp.dat
grid              : ethanol_1_grid.dat, ethanol_2_grid.dat
vdw_radii         : None

[hyperbolic.restraint.options]
weight            : 1.0, 1.0
restraint         : True
resp_a            : 0.001
resp_b            : 0.1
ihfree            : False
toler             : 1e-5
max_it            : 25 

[constraints]
# fixed charges: atom_number (in xyz file) = partial_charge,
constraint_charge : 3 = -0.63087078,
                    4 = 0.377965

#groups of equivalent atoms -> group_n = atom_number list,
equivalent_groups : group_1 = 7 8 9,
                    group_2 = 5 6

[qm.options]
method_esp    : None
basis_esp     : None
formal_charge : None
multiplicity  : None