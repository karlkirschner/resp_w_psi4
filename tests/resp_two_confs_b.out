Electrostatic potential parameters
    van der Waals radii (Angstrom):
                                      C = 1.500
                                      O = 1.400
                                      H = 1.200
    VDW scale factors:                1.4 1.6 1.8 2.0
    VDW point density:                1.0
    ESP method:                       None
    ESP basis set:                    None

Grid information
    Quantum ESP File(s):
                                      ethanol_1_esp.dat
                                      ethanol_2_esp.dat

    Grid Points File(s) (# of points):
                                      ethanol_1_grid.dat (520)
                                      ethanol_2_grid.dat (517)

Constraints
    Charge constraints:
                                      Atom 3 = -0.63087078
                                      Atom 4 = 0.377965

    Equivalent charges on atoms (group = atom numbers):
                                      group_1 = 7 8 9 
                                      group_2 = 5 6 

Restraint
    ihfree:                           False
    resp_a:                           0.0010
    resp_b:                           0.1000

Fit
    Electrostatic Potential Charges:
        Center  Symbol        esp         resp        
          1        C      -0.26320349 -0.07985543
          2        C       0.30502817  0.25393230
          3        O      -0.63087078 -0.63087078
          4        H       0.37796500  0.37796500
          5        H      -0.01339645 -0.00771573
          6        H      -0.01339645 -0.00771573
          7        H       0.07929133  0.03142012
          8        H       0.07929133  0.03142012
          9        H       0.07929133  0.03142012

        Total Charge:      0.00000000  0.00000000
